samples/starterKit/skill/MainStateMachine.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+// Include the state machine module and the replyWith function
+const Voxa = require('voxa');
+const views = require('./views');
+const variables = require('./variables');
+const states = require('./states');
+const voxaAma = require('voxa-amazon-mobile-analytics');
+const config = require('../config');
+
+const skill = new Voxa({ variables, views });
+
+states.register(skill);
+voxaAma(skill, config.ama);
```

Comment:
do we have examples of how this config looks like?

Reply:
@armonge in the README:
```
const amaConfig = {
  appId: 'appId',
  appTitle: 'appTitle',
  make: 'Amazon',
  platform: 'Alexa',
  cognitoIdentityPoolId: 'cognitoIdentityPoolId',
  suppressSending: false, // A flag to supress sending hits. Useful while developing on the skill
};
```